common/test/acceptance/pages/lms/course_wiki.py

```diff
@@ -79,3 +97,41 @@ def save_wiki_content(self):
         """
         self.q(css='button[name="save"]').click()
         self.wait_for_element_presence('.alert-success', 'wait for the article to be saved')
+
+
+class CourseWikiChangesPage(CourseWikiSubviewPage):
+    """
+    Course wiki changes page.
+    """
+
+    def is_browser_on_page(self):
+        """
+        Return if the browser is on the history page.
+        """
+        return self.q(css='section.history').present
+
+    @property
+    def url_path(self):
+        """
+        Construct a URL to the page within the course.
+        """
+        return "/wiki/" + self.article_name + "/_history"
+
+
+class CourseWikiChildrenPage(CourseWikiSubviewPage):
+    """
+    Course wiki "All Children" page.
+    """
+
+    def is_browser_on_page(self):
+        """
+        Return if the browser is on the wiki children page (which contains a search widget).
+        """
+        return self.q(css='.form-search').present
```

Comment:
Unfortunately there's not a good top-level element to look for on this particular view.